### PR TITLE
Standards associations

### PR DESCRIPTION
**Pull request by Erin007**
Continuation of work in #32628.  This uses the metadata information returned from Curriculum Builder to associate Code Studio Stages with the correct Code Studio Standards. 

When the import is successful: 
successful-import

When there is lesson information from Curriculum Builder that doesn't match a Code Studio Stage or when there is standards information from Curriculum Builder that doesn't match Standards on Code Studio: 
Screen Shot 2020-01-16 at 4 06 08 PM

### CODE REVIEW

File: dashboard/app/controllers/admin_standards_controller.rb [hunk 1]
```diff
@@ -2,10 +2,52 @@ class AdminStandardsController < ApplicationController
...
   before_action :require_admin
 
-  def import_standards
+  def index
     @scripts_for_standards = Script.scripts_for_standards
   end
 
-  def update_standards
+  def import_standards
+    script = Script.find_by_name(params["stage_name"])
```
Comment (davidsbailey) [inline dashboard/app/controllers/admin_standards_controller.rb]: sorry for missing this earlier, but we should really confine the erroneous use of the term "stage_name" to as few places as possible. It's ok if we don't want to remove it from the CB database, but can we please change this to "unit_name" for anything in dashboard, as well as any APIs that are sending stuff between CB and dashboard?

**Review by davidsbailey — APPROVED** [review]
just one gripe.

File: dashboard/app/controllers/admin_standards_controller.rb (hunk 1 above)
Comment (Erin007) [reply dashboard/app/controllers/admin_standards_controller.rb]: Yep! I wasn't sure if it was better to stay consistent with CB or not, but happy to update.

Comment (dmcavoy) [reply dashboard/app/controllers/admin_standards_controller.rb]: wouldn't stage name be the same as lesson name not unit name?

**Review by dmcavoy — APPROVED** [review]

File: bin/import_standards_from_csv [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        if loaded.update!(parsed)
+          updated += 1
+        end
+      end
+    end
+  end
+  puts "Created #{created} standards, updated #{updated}"
+end
+
+main(ARGV[0])
```
Comment (clareconstantine) [inline bin/import_standards_from_csv]: I have a separate PR for this script with some feedback I need to incorporate still - let's keep this bit separate for now (not sure if you meant to include it here)

**Review by clareconstantine — APPROVED** [review]

File: dashboard/app/controllers/admin_standards_controller.rb (hunk 1 above)
Comment (davidsbailey) [reply dashboard/app/controllers/admin_standards_controller.rb]: it should be... but `stage_name` on the Unit model in CB is poorly named and is actually the code studio unit name.

File: bin/import_standards_from_csv (hunk 2 above)
Comment (davidsbailey) [reply bin/import_standards_from_csv]: Not sure if this is in response to my comments, but if it is then yes it seems fine to address those in a separate PR.

File: dashboard/app/controllers/admin_standards_controller.rb (hunk 1 above)
Comment (Erin007) [reply dashboard/app/controllers/admin_standards_controller.rb]: https://github.com/code-dot-org/curriculumbuilder/pull/242 <- updated in CurriculumBuilder here; I'll follow up with the needed Code Studio changes once that PR gets merged.

**Pull request merged**